src/Employer/Employer.Web/Controllers/DashboardController.cs

```diff
@@ -31,6 +31,12 @@ public async Task<IActionResult> Dashboard([FromRoute]string employerAccountId,[
 
             var vm = await _orchestrator.GetDashboardViewModelAsync(employerAccountId, filter, page);
 
+            vm.FromEmployer = bool.Parse(fromEmployer);
```

Comment:
I think this would be less likely to break if someone is hacking about constructing their own urls.
`vm.FromEmployer = fromEmployer != "false";`